packages/react-native-sdk/src/components/ActiveCall.tsx

```diff
@@ -78,6 +83,7 @@ const InnerActiveCall = (props: ActiveCallProps) => {
       <View onLayout={onLayout} style={styles.callControlsWrapper}>
         <CallControlsView onHangupCall={onHangupCall} />
       </View>
+      {!disableDefaultSorter && <ActiveCallDefaultSorter />}
```

Comment:
If a participant doesn't want to opt for default sorting, are we asking them to design the entire `ActiveCall` component? 🤔